chapter-05/recipe-08/c-example-3.5/CMakeLists.txt

```diff
@@ -25,14 +26,19 @@ int main(int argc, char * argv[]) {
   return 0;
 }
   ")
+
 set(CMAKE_REQUIRED_INCLUDES ${UUID_INCLUDE_DIRS})
 set(CMAKE_REQUIRED_LIBRARIES ${UUID_LIBRARIES})
 check_c_source_runs("${_test_uuid}" _runs)
+unset(CMAKE_REQUIRED_INCLUDES)
+unset(CMAKE_REQUIRED_LIBRARIES)
+
 if(NOT _runs)
   message(FATAL_ERROR "Cannot run a simple C executable using libuuid!")
 endif()
 
 add_executable(use-uuid use-uuid.cpp)
```

Comment:
The source code is C++ but the example folder says C. Is this intended?

Reply:
Yes, it was much clearer after reading the text. I am merging this one in the hope that we get appveyor to run on `master` Can you look into Appveyor settings to see if anything changed?

Reply:
Ah, and can you also delete the branch?